.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -15,8 +15,8 @@ labels: bug
 ## Dialect
 
 ## Version
-Include the output of `sqlfluff --version` along with your Python version.
-If you are using dbt then please additionally include the dbt version.
+<!-- Include the output of `sqlfluff --version` along with your Python version.
```

Comment:
I wonder if they'll include the right version information without these explicit instructions. (Especially dbt version.)

Reply:
It will show to the raiser (as they see unrendered version) just won’t appear in actual PR when rendered.

So I’m happy with this. We do same with comments at the top.

Reply:
Yeah you can always see the comments when submitting or editing the issue it just hides it when submitted
![image](https://user-images.githubusercontent.com/80432516/150415573-1eef67ac-dd8a-4712-8af2-d02f82d59d1c.png)

Reply:
Sounds good, thanks!